clients/src/main/resources/common/message/ProduceRequest.json

```diff
@@ -35,7 +35,10 @@
   // Version 9 enables flexible versions.
   //
   // Version 10 is the same as version 9 (KIP-951).
-  "validVersions": "0-10",
+  //
+  // Version 11 is schematically same as Version 10. It will only enable the server to know that
```

Comment:
For now, let's just say Version 11 is the same as 10. We may decide to do a separate bump for the implicit add partitions to transaction.